submissions/yaripey/dom-task/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="css/main.css">
+  <script src="js/main.js"></script>
+</head>
+
+
+<body>
+
+  <header>
+    <a href="#" class="toggle"></a>
```

Comment:
It's better to make this element as `button`. Semantically it will be more correct

Reply:
Made it a button. Also added some styles to compensate button's default estyles.